source/_posts/2016-12-03-remote-websockets-sonarr.markdown

```diff
@@ -212,7 +212,6 @@ Experiencing issues introduced by this release? Please report them in our [issue
 [gpslogger]: /integrations/gpslogger
 [harmony]: /integrations/harmony
 [homematic]: /integrations/homematic/
-[hook]: /integrations/hook
```

Comment:
Just removing the Markdown reference is not enough, you have to make sure the link that refers to this is also removed.

E.g., if there is written on that page: `Nice little [Hook][hook] integration` it becomes -> `Nice little Hook integration`.

Reply:
Hope I got all of them now 😅